### PR TITLE
Docs: fix indentation and update resources fields

### PR DESCRIPTION
**Pull request by zhujian7**
Fix indentation and update resources fiedls to make the sample executable.

### CODE REVIEW

File: docs/using.md [hunk 1]
```diff
@@ -186,18 +186,19 @@ Use the following example to understand the syntax and strucutre of a Git Resour
...
+      inputs:
+        resourcesVersion:
+        - resourceRef:
+            name: wizzbang-git
+            apiVersion: HEAD
+      outputs:
+        resources:
+        - name: builtImage
+          type: image
+      # serviceAccount: build-bot
```
Comment (pivotal-nader-ziada) [inline docs/using.md]: if you want to add serviceAccount here can you add not make commented out and put a comment that it is optional

Comment (zhujian7) [reply docs/using.md]: OK, i will fix it.

Comment (zhujian7) [reply docs/using.md]: I have fixed it, thank you.

**Comment by bobcatfish** [comment]
Nice, thanks for fixing this @zhujian7 !!

**Pull request merged**